src/Modules/Unix/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1

```diff
@@ -1,32 +1,34 @@
 @{
-GUID="1DA87E53-152B-403E-98DC-74D7B4D63D59"
-Author="PowerShell"
-CompanyName="Microsoft Corporation"
-Copyright="Copyright (c) Microsoft Corporation. All rights reserved."
-ModuleVersion="6.1.0.0"
+GUID = "1DA87E53-152B-403E-98DC-74D7B4D63D59"
+Author = "PowerShell"
+CompanyName = "Microsoft Corporation"
+Copyright = "Copyright (c) Microsoft Corporation. All rights reserved."
+ModuleVersion = "6.1.0.0"
 CompatiblePSEditions = @("Core")
-PowerShellVersion="3.0"
-CmdletsToExport= "Format-List", "Format-Custom", "Format-Table", "Format-Wide",
-    "Out-File", "Out-String", "Get-FormatData", "Export-FormatData", "ConvertFrom-Json", "ConvertTo-Json",
-    "Invoke-RestMethod", "Invoke-WebRequest", "Register-ObjectEvent", "Register-EngineEvent",
-    "Wait-Event", "Get-Event", "Remove-Event", "Get-EventSubscriber", "Unregister-Event", "New-Guid",
-    "New-Event", "Add-Member", "Add-Type", "Compare-Object", "ConvertTo-Html", "ConvertFrom-StringData",
-    "Export-Csv", "Import-Csv", "ConvertTo-Csv", "ConvertFrom-Csv", "Export-Alias", "Invoke-Expression",
-    "Get-Alias", "Get-Culture", "Get-Date", "Get-Host", "Get-Member", "Get-Random", "Get-UICulture",
-    "Get-Unique", "Export-PSSession", "Import-PSSession", "Import-Alias", "Import-LocalizedData",
-    "Join-String", "Select-String", "Measure-Object", "New-Alias", "New-TimeSpan", "Read-Host", "Set-Alias", "Set-Date",
-    "Start-Sleep", "Tee-Object", "Measure-Command", "Update-TypeData", "Update-FormatData",
-    "Remove-TypeData", "Get-TypeData", "Write-Host", "Write-Progress", "New-Object", "Select-Object",
-    "Group-Object", "Sort-Object", "Get-Variable", "New-Variable", "Set-Variable", "Remove-Variable",
-    "Clear-Variable", "Export-Clixml", "Import-Clixml", "Import-PowerShellDataFile", "ConvertTo-Xml", "Select-Xml", "Write-Debug",
-    "Write-Verbose", "Write-Warning", "Write-Error", "Write-Information", "Write-Output", "Set-PSBreakpoint",
-    "Get-PSBreakpoint", "Remove-PSBreakpoint", "Enable-PSBreakpoint", "Disable-PSBreakpoint", "Get-PSCallStack",
-    "Send-MailMessage", "Get-TraceSource", "Set-TraceSource", "Trace-Command", "Get-FileHash",
-    "Get-Runspace", "Debug-Runspace", "Enable-RunspaceDebug", "Disable-RunspaceDebug",
-    "Get-RunspaceDebug", "Wait-Debugger" , "Get-Uptime", "New-TemporaryFile", "Get-Verb", "Format-Hex",
-    "Test-Json", "Remove-Alias", "ConvertFrom-Markdown", "Show-Markdown", "Set-MarkdownOption", "Get-MarkdownOption"
-FunctionsToExport= "Import-PowerShellDataFile"
-AliasesToExport= "fhx"
-NestedModules="Microsoft.PowerShell.Commands.Utility.dll","Microsoft.PowerShell.Utility.psm1"
+PowerShellVersion = "3.0"
+CmdletsToExport = @(
```

Comment:
When I see the list, I always think about alphabetical sorting.

Reply:
It is sorted alphabetically, on the noun part of the command name, as you can see `Alias`, `Clixml`, `Command`, `Csv` ... I think related commands should be grouped together. This is not perfect though, because for `Format-*` commands, they should be grouped based on the verb part ...

Reply:
Are you suggesting list all commands as a 1-column list like in `management.psd1`? There are 57 cmdlets in `management.psd1` but `108` in `utility.psd1`. The 1-column list would be too long.

Reply:
The 1-columns sort looks good to me.

Reply:
I think it's probably better to group related cmdlets together in the same row, so the list will be like this:
```
'Get-Alias', 'Set-Alias', 'Remove-Alias',
'Import-Csv', 'Export-Csv',
...
```
But I will deffer this change to a separate follow-up PR.